cves/kernel/CVE-2017-15274.yml

```diff
@@ -251,8 +249,10 @@ i18n:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
+    The cause for the vulnerability has to do with the values of pointers, which
+    would not be affected by internationalization features.
 sandbox:
   question: |
     Did this vulnerability violate a sandboxing feature that the system
```

Comment:
The questions below until the CWE seems to be missing answers--this might also be the reason why the build is failing.